include/assimp/BaseImporter.h

```diff
@@ -274,6 +274,16 @@ class ASSIMP_API BaseImporter {
             const char *ext1 = nullptr,
```

Comment:
Not without a major version bump.

Reply:
Not without a major version bump. Just add an issue report as techdept and we can do this at the next major-report. From ym point of view this could be when the new animation system is not experimental any more.